evaluation.py

```diff
@@ -13,17 +13,18 @@
 ANGLE_IDX = 5
 
 
-def evaluate_model(generator, experiment, test_set, batch_size, batch_num, parametres, device, scaler, step):
+def evaluate_model(generator, experiment, test_set, batch_size, batch_num, parametres, device, scaler, step,
+                   gan_test_ratio):
     features = PTCL_FEATURES.copy()
     del features[ANGLE_IDX]
     features += DIJET_SYSTEM_FEATURES
     predictions = []
+    generator.eval()
     for _ in tqdm(range(batch_num), desc='evaluation', position=0, leave=True):
         x_noise = torch.randn((batch_size, parametres.gan_noise_size), device=device)
         predictions.append(generator(x_noise).cpu().detach().numpy())
-
     predictions_np = np.concatenate(predictions)
-
+    generator.train()
```

Comment:
It is important to switch in eval mode, but switching back to `.train()` usually do inside training function at every iteration beginning. In this care evaluation function do not care about what's going outside it, as well as train function do not care about what's going outside train.